### ISSUE TITLE
Default build produces godot-cpp.linux.debug.64, but demo example SConstruct is looking for godot-cpp.linux.64

### ISSUE THREAD

**Issue by spongeboburu** [GodotNativeTools/godot-cpp#185]
I was following the tutorial and had a linker problem at the last build step

I built godot-cpp with ```scons platform=linux generate_bindings=yes use_custom_api_file=yes custom_api_file=../api.json```.

then

```
scons platform=linux
...
/bin/ld: cannot find -lgodot-cpp.linux.64
```

What I have:
```
godot-cpp/bin/libgodot-cpp.linux.debug.64.a
```

**Comment by marcosbitetti** [comment]
renamed to libgodot-cpp.linux.64.a solve for me

**Comment by BastiaanOlij** [comment]
Yes, the library being prefixed with lib was changed a little while back when the SConstruct file of godot-cpp was improved a fair bit. Just need to make sure that you adjust the SConstruct file of your project accordingly

**Comment by BastiaanOlij** [comment]
I hope to find some time to go through all the GDNative examples and update them for nativescript 1.1, if anyone else want to help out here, that be great.
